internal/api/handlers.go

```diff
@@ -252,6 +257,105 @@ func DeleteCertificate(env *Environment) http.HandlerFunc {
 	}
 }
 
+// GetUserAccounts returns all users from the database
+func GetUserAccounts(env *Environment) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		users, err := env.DB.RetrieveAllUsers()
+		if err != nil {
+			logErrorAndWriteResponse(err.Error(), http.StatusInternalServerError, w)
+			return
+		}
+		apiUsers := make([]certdb.User, 0, len(users))
+		for _, user := range users {
+			apiUsers = append(apiUsers, certdb.User{
+				ID:          user.ID,
+				Username:    user.Username,
+				Permissions: user.Permissions,
+			})
+		}
+		body, err := json.Marshal(apiUsers)
```

Comment:
This is why we modified the json tags on the structs. Read below on why we set the password to `""`
```suggestion
		for i := range users {
			users[i].Password = ""
		}
		body, err := json.Marshal(users)
```

Reply:
✅